tools/Ensembl-REST/.shed.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Ensembl-REST
+owner: earlhaminst
+description: Ensembl-REST wrappers
+long_description: http://rest.ensembl.org
+remote_repository_url: https://github.com/TGAC/earlham-galaxytools/tree/master/tools/Ensembl-REST
+type: unrestricted
+auto_tool_repositories:
+  name_template: "Ensembl {{ tool_id }}"
+  description_template: "Wrapper for Ensebl-REST {{ tool_name }}."
```

Comment:
What about: `"{{ tool_name }} (from the Ensebl-REST tool suite)"` which would result in:

- "Get Ensembl features by ID (from the Ensebl-REST tool suite)"
- "Get gene tree by Ensembl ID (from the Ensebl-REST tool suite)"
- "Get sequences by Ensembl ID (from the Ensebl-REST tool suite)"